src/components/userGroups/UserGroup.tsx

```diff
@@ -1477,6 +1515,41 @@ function NoUsers({ onClick }) {
                 </button>
             </EmptyState.Button>
         </EmptyState>
+       
+
+    )
+}
+function NoSSO({ onClick }) {
+    return (
+        
+
+        <EmptyState>
+            <EmptyState.Image>
+                <img src={EmptyImage} alt="so empty" />
+            </EmptyState.Image>
+            <EmptyState.Title>
+                <h4>No users</h4>
+            </EmptyState.Title>
+            <EmptyState.Subtitle className='w-320 '>Add users and assign group or direct permissions
+                <InfoColourBar
+                    message={`Devtron uses Single Sign-On (SSO) to enable one-click login. Please set up an SSO login service before adding users.Go to SSO login services`}
+                    classname="error_bar m-t-17 text-align-left info-colour-bar svg  "
+                    linkText ={"Go to SSO login services"}
+                    redirectLink={"/global-config/login-service"}
```

Comment:
Is this Redirect link working??

Reply:
Yes. Working fine for me, tested on local.